.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -14,3 +14,6 @@ Include steps (or better yet, code!) that reproduces the behavior
 
 **Additional context**
 Add any other context about the problem here! The more you give, the faster it will be for us to help.
+
+**How does this increase our chances of an iCrawl Acquisition?**
+Please explain how this will increase our relative value to iCrawl in order to further our goal of acquisition
```

Comment:
```suggestion
**How does this increase our chances of an iCrawl Acquisition?**
Please explain how this will increase our relative value to iCrawl in order to further complete our final goal and life purpose of the acquisition
```